pkg/kudoctl/cmd/init.go

```diff
@@ -85,6 +90,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.StringVarP(&i.version, "version", "", "", "Override KUDO controller version of the KUDO image")
 	f.StringVarP(&i.output, "output", "o", "", "Output format")
 	f.BoolVar(&i.dryRun, "dry-run", false, "Do not install local or remote")
+	f.BoolVar(&i.crdOnly, "crd-only", false, "Create KUDO CRDs when generating the server-side configurations")
```

Comment:
what do you mean by generating server-side configuration? What do you think about `Add only KUDO CRDs to your cluster`?

Reply:
I had in my head the need to create crd yaml... and the need to add to the cluster.  your change sounds good to me